packages/console/src/components/AppContent/components/UserInfo/index.module.scss

```diff
@@ -0,0 +1,50 @@
+@use '@/scss/underscore' as _;
+
+.container {
+  display: flex;
+  align-items: center;
+  padding: _.unit(2);
+  margin-left: _.unit(4);
+  border-radius: 8px;
+  transition: background-color 0.2s ease-in-out;
+  user-select: none;
+  cursor: pointer;
+
+  &:hover,
+  &.active {
+    background-color: var(--color-surface-5);
+  }
+
+  img {
+    width: 36px;
+    height: 36px;
+    margin-right: 8px;
+    border-radius: 6px;
```

Comment:
Use `_.unit` instead?

Reply:
It's ok to use px values on border radius
Unit is for paddings and margins

Reply:
how about margin-right above. Could use _.unit.

Reply:
Sorry, I commented on the wrong code line.
see the line 21: `margin-right: 8px`

Reply:
Oh, I didn't realize that. Haha, thanks